### DISCUSSION TITLE
it exceeds 250M - plotly + pandas + numpy = 274Md

### DISCUSSION THREAD

**Discussion by mateusmsantin** [vercel/community#3796] [Help]
### Summary

I'm trying to run a Django application that uses the following requirements below. I put the size of the package to the side and it is noticed that it exceeds 250M, so I believe that I get the message after the deploy.
"Error: A serverless function exceeded the maximum uncompressed size of 250 MB.: https://vercel.link/serverless-function-size"
Am I correct about the error and dependencies or is there another reason for the error?
I appreciate any comments on this.

**requirements.txt** 

**144M	                    site-packages/plotly
 70M	                    site-packages/pandas
 60M	                    site-packages/numpy**
 38M	                    site-packages/django
 16M	                    site-packages/pip
9.8M	                    site-packages/lxml
9.5M	                    site-packages/fonttools
6.5M	                    site-packages/setuptools
5.0M	                    site-packages/psycopg2
3.9M	                    site-packages/virtualenv
2.6M	                    site-packages/tzdata
2.6M	                    site-packages/pytz
2.4M	                    site-packages/joblib
2.4M	                    site-packages/aiohttp
1.8M	                    site-packages/psycopg
1.7M	                    site-packages/distlib
1.2M	                    site-packages/jinja2
1.1M	                            site-packages/websockets
968K	                    site-packages/urllib3
924K	                    site-packages/anyio
888K	                    site-packages/pyparsing
880K	                    site-packages/contourpy
852K	                    site-packages/openai
832K	                    site-packages/click
740K	                    site-packages/starlette
612K	                    site-packages/uvicorn
576K	                    site-packages/wheel
556K	                    site-packages/htmltools
548K	                    site-packages/tqdm
528K	                    site-packages/idna
524K	                    site-packages/h11
472K	                    site-packages/requests
400K	                    site-packages/sqlparse
400K	                    site-packages/immutables
364K	                    site-packages/packaging
304K	                    site-packages/certifi
272K	                    site-packages/yarl
248K	                    site-packages/kiwisolver
188K	                    site-packages/multidict
176K	                    site-packages/tenacity
172K	                    site-packages/asgiref
168K	                    site-packages/platformdirs
168K	                    site-packages/markupsafe
152K	                    site-packages/frozenlist
 84K	                    site-packages/filelock
 72K	                    site-packages/mdurl
 52K	                    site-packages/attrs
 40K	                    site-packages/sniffio
 20K	                    site-packages/contextvars
 12K	                            site-packages/aiosignal

### Steps to Reproduce

Run vecel build whit this requirements.txt

**Comment by felipegambini** [comment]
I'm getting the same error with the same libraries, I have researched a lot and have not found any alternative way. It has been quite disappointing.

Are you still trying or have you given up?
